Assets/HoloToolkit/UI/Scripts/UsabilityUtilities.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// A helper class for making applications more usable across different devices.
+    /// </summary>
+    public static class UsabilityUtilities
```

Comment:
Thanks for the feedback.  Can you say a bit more about what the caller would pass in for targetPPD and how they would choose that?

Ultimately, we felt we wanted something similar to ["device-independent pixels"](https://en.wikipedia.org/wiki/Device-independent_pixel) in 2D UI frameworks, so that most creators can design for whatever device they physically have access to, and things will "just work" on devices they've never physically tested.